hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java

```diff
@@ -77,6 +81,31 @@ public void treasuryDisbursement(long amount, String accountName) {
         assertNotNull(networkTransactionResponse.getReceipt());
     }
 
+    @Given("I send {long} tℏ from {string} to {string} alias not present in the network")
```

Comment:
should just send fund from the client operator to the new alias account, this will reduce the complexity of the test

```suggestion
    @Given("I send {long} tℏ to {string} alias not present in the network")
```

Reply:
fixed